app/src/editor/canvas/tests/variadic.test.js

```diff
@@ -581,5 +581,61 @@ describe('Variadic Port Handling', () => {
             // test server accepts state
             expect(State.updateCanvas(await Services.create(canvas))).toMatchCanvas(canvas)
         })
+
+        it('permits renaming outputs', async () => {
+            let canvas = State.emptyCanvas()
+            canvas = State.addModule(canvas, await loadModuleDefinition('ConstantText'))
+            canvas = State.addModule(canvas, await loadModuleDefinition('PassThrough'))
+            let [
+                constantText1,
+                passThrough1,
+            ] = canvas.modules
+
+            // connect constantText1.out to passThrough.in1
+            canvas = State.updateCanvas(State.connectPorts(canvas, constantText1.outputs[0].id, passThrough1.inputs[0].id))
+
+            // update variables
+            ;[ // eslint-disable-line semi-style
```

Comment:
👀 What's happening here?

Reply:
if you don't use destructuring, eslint complains that you didn't use destructuring but to use destructuring you need to have a semi before the `[`, if you put it at the start of the line, eslint complains that it's not on the end of the previous line, if you put it on the end of the previous line, eslint complains that the semi exists.

Reply:
Ah, ok!